### DISCUSSION TITLE
True / False fields are not translated to german in filter webpart

### DISCUSSION THREAD

**Discussion by StefanWeyer** [microsoft-search/pnp-modern-search#2838] [Ideas]
Whe we use a true or false field on a german page, the filter webpart still shows true / false instead of Ja/Nein.

**Comment by kasperbolarsen** [comment]
Which version?

**Comment by StefanWeyer** [comment]
4.8.0

**Comment by wobba** [comment]
@StefanWeyer I assume these are values on results, and we don't control those values. I'll tag as an enhancement to see if we can solve this somehow.

**Reply by StefanWeyer** [reply]
Thank you 👍  - I don't understand all of this code in the filter webpart:

```html
{{#if filter.isMulti}}
	<pnp-filtermultiselect 
		data-instance-id="{{@root.instanceId}}" 
		data-filter-name="{{filter.filterName}}" 
		data-apply-disabled="{{#if filter.canApply}}false{{else}}true{{/if}}" 
		data-clear-disabled="{{#if filter.canClear}}false{{else}}true{{/if}}">
	</pnp-filtermultiselect>
{{/if}}
```

This might be the part. If it would take the value it should be Ja/Nein because we have a Regional and Language set to German/Switzerland. Or is it a internal value?

As I'm not aware of all the things we can do is there a possibility to make that to Ja/Nein with some changes on the code with If etc.? That would help as well. Thank you for your support! :-)

**Reply by StefanWeyer** [reply]
Oh, seems that it has removed most of the code here, sorry I pasted the whole passage

**Reply by wobba** [reply]
Fixed your template.. add three back-ticks before and after the block to render as code (markdown syntax).

The issue is that returned values are boolean programatic values, which are not localized. In programming languages they are returned as `true` or `false`. And I assume the managed property you are using is of type Yes/No?

**Reply by StefanWeyer** [reply]
Thank you for your support @wobba :-) - Yes it is a Yes/No Field.
What do you mean with three back-ticks before and after the block to render? 

```html
<template id="collapsible-footer">
	{{#isnt filter.selectedTemplate 'DateRangeFilterTemplate'}}
			{{#eq filter.values.length 0}}
				<div class="filter--message">
					{{@root.strings.FilterNoValuesMessage}}
				</div>
			{{else}}
				{{#if filter.isMulti}}
					<pnp-filtermultiselect 
						data-instance-id="{{@root.instanceId}}" 
						data-filter-name="{{filter.filterName}}" 
						data-apply-disabled="{{#if filter.canApply}}false{{else}}true{{/if}}" 
						data-clear-disabled="{{#if filter.canClear}}false{{else}}true{{/if}}">
					</pnp-filtermultiselect>
				{{/if}}
			{{/eq}}
	{{/isnt}}
</template>
```

That doesn't work really. Sorry, I'm not really proofed at this kind of code :-D - Thanks for help!

**Reply by wobba** [reply]
Formatted it for you :) To render code blocks on github issues. I have not looked at your template, but you might be able to add something to look at the values and translate them. A bit further up on the `<pnp-filtercheckbox>` parts.

![image](https://user-images.githubusercontent.com/51104/224811305-9f0b83b9-6d31-40e6-94fd-d9f1e44e4da6.png)
